the-internet/pom.xml

```diff
@@ -31,9 +31,9 @@
 			<version>7.1.0</version>
 			<scope>test</scope>
 		</dependency>
-		
+
```

Comment:
public class TheInternet:
Configurations (browserType, etc) should be read from an external file.